src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -68,48 +68,54 @@ <h4 class="panel-title border-bottom-small" translate>
         <h4 class="panel-title border-bottom-small" translate>
           {{'GIFT_AMOUNT'}}
         </h4>
-        <div class="panel-body pt0">
+        <div class="panel-body p0">
           <div ng-if="$ctrl.useSuggestedAmounts">
-            <div class="radio radio-custom-amount"
-                 ng-repeat="suggested in $ctrl.suggestedAmounts | orderBy:'order'">
-              <label>
-                <input name="suggestedAmount"
-                       type="radio"
-                       ng-click="$ctrl.changeAmount(suggested.amount)"
-                       ng-checked="!$ctrl.customInputActive && $ctrl.itemConfig.AMOUNT === suggested.amount" />
-                {{$ctrl.suggestedAmount(suggested.amount)}} {{suggested.label}}
-              </label>
-            </div>
-            <div class="radio radio-custom-amount form-inline" ng-class="{'has-error': ($ctrl.itemConfigForm.amount | showErrors)}">
-              <div class="form-group">
+            <div class="button-group">
+              <div 
+                  role="button"
+                  class="suggested-button"
+                  ng-click="$ctrl.changeAmount(suggested.amount)"
+                  ng-class=" {'active': !$ctrl.customInputActive && $ctrl.itemConfig.AMOUNT === suggested.amount}"
+                  ng-repeat="suggested in $ctrl.suggestedAmounts | orderBy:'order'">
+                  <div class="amount-box">
+                    {{$ctrl.suggestedAmount(suggested.amount)}}
+                  </div>
+                  <div class="label-text">
+                    {{suggested.label}}
+                  </div>
+              </div>
+              <div class="custom-amount-button" ng-class="{'has-error': ($ctrl.itemConfigForm.amount | showErrors)}">
+                <div class="input-prepend">
+                  <span class="input-prepend-text">$</span>
+                </div>
                 <label>
-                  <input name="suggestedAmount"
-                         type="radio"
-                         ng-checked="$ctrl.customInputActive" />
-                  <input class="form-control form-control-subtle"
-                         name="amount"
-                         type="text"
-                         step="1"
-                         tabindex="-1"
-                         ng-model="$ctrl.customAmount"
-                         ng-change="$ctrl.changeCustomAmount($ctrl.customAmount)"
-                         ng-focus="$ctrl.customInputActive = true;"
-                         ng-required="$ctrl.customInputActive"
-                         placeholder="{{'OTHER_PLACEHOLDER' | translate}}" />
+                  <input 
+                    name="suggestedAmount"
+                    type="radio"
+                    ng-checked="$ctrl.customInputActive" />
+                  <input
+                    name="amount"
+                    type="text"
+                    step="1"
+                    tabindex="-1"
```

Comment:
@wrandall22 was there a reason why we wanted to prevent tab navigation specifically for a custom amount input?

Reply:
It looks like it has been that way from the [beginning](https://github.com/CruGlobal/give-web/commit/53a627310a50165a10dd2c93da648a03e35e8050#diff-8be251bc10d103c8d03b3b4b880cd93b2fa17b86eb51b65106eb7ea0edfe0ddf). I don't see a reason given in that commit. Perhaps it was defaulting to the textbox instead of the first radio button without it?

Reply:
Initially this seems to have been functioning inside a modal which you wouldn't want to have present in the tabindex. So I can see why the attribute default being set to -1 makes sense. I'm wondering if there was or still could be some JS that is handling the focus trap and it's changing that attribute or setting it "".

Reply:
This is still in a modal on standard checkout, just not branded checkout.

Reply:
Oh I see. As of now, a user can tab into it and navigate with arrows. The tabindex threw me off because I assumed it would prevent the ability to focus on the form with tab.